identity-service/sequelize/migrations/20210218204013-create-recaptcha-scores.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('RecaptchaScores', {
+      id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      walletAddress: {
```

Comment:
we should index/key on wallet address so lookups are fast (this table will be huge)